### ISSUE TITLE
Dead links in rtd docs

### ISSUE THREAD

**Issue by prisae** [Statoil/pylops#9]
The "Improve this page" links on read the docs are dead.

---
Example:

![2018-12-22_01](https://user-images.githubusercontent.com/8020943/50380782-8b820500-0638-11e9-94f9-6d4043f08b4f.jpg)

---
Leads to:

![2018-12-22_02](https://user-images.githubusercontent.com/8020943/50380781-8ae96e80-0638-11e9-9a47-c71987a70006.jpg)
---

So it links to 
https://github.com/pylops/edit/master/docs/installation.rst

instead to
https://github.com/Statoil/pylops/edit/master/docs/source/index.rst
or simply
https://github.com/Statoil/pylops/blob/master/docs/source/index.rst

I tried to find the source of the problem in the `conf.py` and the `_templates/breadcrumbs.html` files, but I couldn't locate the problem.

**Comment by mrava87** [comment]
Thanks for noticing this. There was definitely a 'source' missing in the path, which I added by changing 'doc_path': 'docs' into 'doc_path': 'docs/source' in conf.py file. But I don't get why Statoil was ignored despite being 'github_repo': 'Statoil/pylops', in conf.py file. I have changed conf.py and breadcrumbs.html in https://github.com/mrava87/pylops/commit/9684bfd954dc2608f2d526794cad8dbe43c46baf, hopefully this will fix it.

This is how it looks now in my local documentation (https://github.com/Statoil/pylops/blob/master/docs/source/index.rst) and link works, lets see when I build it again on readthedocs if things are fixed :)
